client/src/store/stores/interaction.ts

```diff
@@ -60,28 +73,62 @@ export default class InteractionStore {
   }
 
   @action.bound
-  applyLinkChange(link: Link, change: StateChange) {
-    if (change === StateChange.Deleted) {
-      return this.deleteLink(link);
+  applyLinkChange(hubbleLink: HubbleLink, change: StateChange) {
+    switch (change) {
+      case StateChange.Deleted: {
+        return this.deleteLink(hubbleLink);
+      }
+      // TODO: handle all cases properly
+      case StateChange.Added:
+      case StateChange.Modified:
+      case StateChange.Exists: {
+        return this.addLink(hubbleLink);
+      }
+      default: {
```

Comment:
Redundant three lines L87:89, there is no any other options different from four above